website/static/js/pages/project-settings-page.js

```diff
@@ -147,6 +151,34 @@ $(document).ready(function() {
                 }
             }
         });
+        };
+        // unchecked warning adopted from profile-settings-addons-page.js
+        if(unchecked.length > 0) {
+            var uncheckedText = $.map(unchecked, function(el){
+                return ['<li>', $(el).closest('label').text().trim(), '</li>'].join('');
+            }).join('');
+            uncheckedText = ['<ul>', uncheckedText, '</ul>'].join('');
+            bootbox.confirm({
+                title: 'Are you sure you want to remove the add-ons you have deselected? ',
+                message: uncheckedText,
+                callback: function(result) {
+                    if (result) {
+                        submit();
```

Comment:
I like that you separated the submit logic from this UI; it helps make the code more readable. Thanks!